mediawiki-containers

```diff
@@ -175,7 +188,11 @@ install() {
     check_out_mediawiki_containers
 
     cd $BASEDIR
-    
+
+    if [ "$platform" != "debian" -a "$platform" != "ubuntu" ]; then
```

Comment:
We could perhaps check this right at the start of the install command.

How is the install command getting past this on OS X?